streaming/src/event_service.cc

```diff
@@ -76,11 +87,9 @@ bool EventQueue::Get(Event &evt) {
 
 Event EventQueue::PopAndGet() {
   std::unique_lock<std::mutex> lock(ring_buffer_mutex_);
-  while (Empty() && is_freezed_) {
-    no_empty_cv_.wait(lock);
-  }
-  if (!is_freezed_) {
-    // Return error event if queue is freezed.
+  WaitFor(lock);
```

Comment:
Why are there streaming changes in this pr?

Reply:
I think this is my wrong operation about git.
I am so sorry.
But this code has been merged into master so maybe we can ignore it.

Reply:
Can you merge the latest master, so that it can be removed?